### PR TITLE
bump(main/iperf3): 3.16

### PR DESCRIPTION
**Pull request by tqfx**
Close #19845

### CODE REVIEW

File: packages/iperf3/build.sh [hunk 1]
```diff
@@ -2,9 +2,11 @@ TERMUX_PKG_HOMEPAGE=https://github.com/esnet/iperf
 TERMUX_PKG_DESCRIPTION="TCP, UDP, and SCTP network bandwidth measurement tool"
 TERMUX_PKG_LICENSE="BSD 3-Clause"
 TERMUX_PKG_MAINTAINER="@termux"
-TERMUX_PKG_VERSION=3.15
-TERMUX_PKG_SRCURL=https://fossies.org/linux/privat/iperf-$TERMUX_PKG_VERSION.tar.gz
-TERMUX_PKG_SHA256=bdb77c11f72bce90214883159577fa24412013e62b2083cf5f54391d79b1d8ff
+TERMUX_PKG_VERSION=3.16
+TERMUX_PKG_SRCURL=https://github.com/esnet/iperf/releases/download/$TERMUX_PKG_VERSION/iperf-$TERMUX_PKG_VERSION.tar.gz
+TERMUX_PKG_SHA256=cc740c6bbea104398cc3e466befc515a25896ec85e44a662d5f4a767b9cf713e
+TERMUX_PKG_UPDATE_VERSION_REGEXP="\d+\.\d+"
```
[the remark this answers is not in the document]
Comment (tqfx) [reply packages/iperf3/build.sh:8]: Remove this line `TERMUX_PKG_UPDATE_VERSION_REGEXP="\d+\.\d+"` ?

[the remark this answers is not in the document]
Comment (Biswa96) [reply packages/iperf3/build.sh:8]: I am not sure. The expected version rule should allow any version except alpha/beta/rc ones.

[the remark this answers is not in the document]
Comment (tqfx) [reply packages/iperf3/build.sh:8]: How about `"^\d+.\d+(.\d+)?$"`?

[the remark this answers is not in the document]
Comment (tqfx) [reply packages/iperf3/build.sh:8]: How about "^\d+\.\d+(\.\d+)?$" ?

**Pull request merged**